cmd/createcluster.go

```diff
@@ -191,6 +197,45 @@ func runCreateCluster(w io.Writer, conf clusterConfig) error {
 		}
 	}
 
+	var depositDatas [][]byte
+	for i := 0; i < numDVs; i++ {
+		sk := secrets[i] // Group secret key for this DV
```

Comment:
```suggestion
		sk := secrets[i] // Secret key for this DV
```
can be confusing with Group concepts from cryptography 😅

Reply:
abstract..